src/NUnitEngine/nunit.engine.tests/Services/ExtensionServiceTests.cs

```diff
@@ -135,23 +108,18 @@ public void AllExtensionPointsAreKnown()
         };
 #pragma warning restore 414
 
-        [TestCaseSource("KNOWN_EXTENSIONS")]
+        [TestCaseSource(nameof(KnownExtensions))]
         public void CanListExtensions(string typeName)
         {
-            foreach (IExtensionNode node in _serviceClass.Extensions)
-                if (node.TypeName == typeName)
-                {
-                    Assert.True(node.Enabled);
-                    return;
-                }
-            
-            Assert.Fail("Couldn't find known Extension {0}", typeName);
+            Assert.That(_serviceInterface.Extensions,
```

Comment:
Before we iterated over `_serviceClass.Extensions`. I don't know much about this area, so why this change?

Reply:
In all the refactorings you did in this PR you "translated" a foreach loop, e.g. `foreach (IExtensionNode node in _serviceInterface.Extensions)` into an assertion over the same collection `Assert.That(_serviceInterface.Extensions, ...`. 

But in this test the foreach loop was `foreach (IExtensionNode node in _serviceClass.Extensions)`, but this was translated into `Assert.That(_serviceInterface.Extensions, ...`, so I was wondering why the change from `_serviceClass.Extensions` to `_serviceInterface.Extensions`.

Reply:
https://github.com/nunit/nunit-console/pull/459